app/scripts/OverlayTrack.js

```diff
@@ -299,12 +389,13 @@ class OverlayTrack extends PixiTrack {
               fill,
               stroke,
               outline,
+              options,
             ),
           );
         }
 
         if (op.orientation === '1d-vertical' || op.orientation === '2d') {
-          this.options.extent.forEach((extent) =>
+          this.options.extent.forEach(extent =>
```

Comment:
Which version of prettier are you using? The default (in v2) should be to add parens around anonymous function arguments.

Reply:
Forgot to do `npm i`. Thanks for the catch